cmd/pomerium/main.go

```diff
@@ -213,11 +213,15 @@ func parseOptions(configFile string) (*config.Options, error) {
 	metrics.AddPolicyCountCallback(o.Services, func() int64 {
 		return int64(len(o.Policies))
 	})
-	checksumInt, err := strconv.ParseInt(fmt.Sprintf("0x%s", o.Checksum()), 0, 64)
+
+	float := big.Float{}
+	checksumDec, _, err := float.Parse(fmt.Sprintf("0x%s", o.Checksum()), 0)
+	//checksumInt, err := strconv.ParseFloat(fmt.Sprintf("0x%s", o.Checksum()), 64)
```

Comment:
```
cmd/pomerium/main.go:219:2: commentedOutCode: may want to remove commented-out code (gocritic)
	//checksumInt, err := strconv.ParseFloat(fmt.Sprintf("0x%s", o.Checksum()), 64)
```
Annoying, but the linter is probably right here

Reply:
Thought I had removed this. 👍